src/Page/Home.purs

```diff
@@ -242,7 +242,7 @@ component = H.mkComponent
         [ HH.i
           [ css "ion-pound" ]
           []
-        , HH.text $ "\160" <> tag
+        , HH.text $ "\x160" <> tag
```

Comment:
I am not 100% sure about this one

Reply:
I _think_ that's just a hash -- so we could stick "#" in there directly. I never remember the codes myself.

Reply:
Done